packages/integration-testsuite/src/apolloServerTests.ts

```diff
@@ -209,6 +209,16 @@ export function defineIntegrationTestSuiteApolloServerTests(
     }
     afterEach(stopServer);
 
+    // This test ensures that consumers of this testsuite are using the lockstep
+    // versioned packages of @apollo/server and
+    // @apollo/server-integration-testsuite. If these versions are out of sync,
+    // this test should fail.
```

Comment:
Wonder if the comment and/or test name could be more clear/verbose here. Does saying something like "It looks like you have multiple copies of `@apollo/server` installed in your project. `@apollo/server-integration-testsuite` always installs a copy of `@apollo/server` with a precisely matching version number; have you installed `@apollo/server` and `@apollo/server-integration-testsuite` with non-matching versions?" help? (I don't know how to print custom error messages though... I guess just manually throwing an Error.)

Reply:
Agree, throwing an error here now (but still running the expect for the known-passing case).